layers/+lang/emacs-lisp/packages.el

```diff
@@ -175,6 +177,21 @@
       (spacemacs/set-leader-keys-for-major-mode 'emacs-lisp-mode
         "dm" 'spacemacs/macrostep-transient-state/body))))
 
+(defun emacs-lisp/init-overseer ()
+  (use-package overseer
+    :defer t
+    :init (spacemacs/set-leader-keys-for-major-mode 'emacs-lisp-mode
+            "oa" 'overseer-test
+            "ot" 'overseer-test-run-test
+            "ob" 'overseer-test-this-buffer
```

Comment:
I believe this should be `SPC m t b` -- similarly probably for (some) other keys as well. See more examples in [CONVENTIONS](https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#tests)

Reply:
@tko do you mean as extra bindings?
I can add those for sure for the one that matches

Reply:
@AdrieanKhisbe `,o` is a prefix [reserved for users](https://github.com/syl20bnr/spacemacs/blob/master/doc/CONVENTIONS.org#user-prefix). Please follow @tko 's suggestion.

Reply:
My bad, i didn't know back then. Just read about it after

Reply:
And here it is. just ammened my commit